Gemfile

```diff
@@ -110,7 +109,7 @@ gem "rack-mini-profiler", require: false # If you can't see it you can't make it
 gem "stackprof", require: false
 
 gem "responders", "~> 2.0" # required because of class level respond_to blocks (API v1)
-gem "thor", "0.19.1" # Locking it; http://stackoverflow.com/questions/40986923/meaning-of-expected-string-default-value-for-on-ruby-on-rails
+gem "thor"
```

Comment:
Stack overflow link goes references this issue, which is already closed: https://github.com/erikhuda/thor/issues/538

Reply:
Resolved by #1070